src/tools/teardown-cf-stack.ts

```diff
@@ -181,7 +181,10 @@ const destroyStack = async (
     ).length;
     const numberLeft = resources.length - numberDeleted;
     const resourcesFailed = resources.filter(
-      (r) => r.ResourceStatus === "DELETE_FAILED"
+      (r) =>
+        r.ResourceStatus === "DELETE_FAILED" ||
+        r.ResourceStatus === "CREATE_COMPLETE" ||
+        r.ResourceStatus === "UPDATE_COMPLETE" // TODO: Remove nonsensical "CREATE_COMPLETE" and "UPDATE_COMPLETE" check once CF is fixed.
```

Comment:
Well it worked differently before. After a delete-stack command, all resources were either DELETE_COMPLETE or DELETE_FAILED, which makes sense. Now, after a delete-stack command there are a wild mix of all possible states. Resources that previously would have been deleted are now still existing with their previous state, although they could have been deleted (like queues and such). CF seems to not even try to delete as much as possible any more, but just stops when something could not be deleted.
I would hope that this is not desired behaviour as it might require multiple re-runs to delete a stack manually, always manually deleting those resources that led to aborting in that run...